### PR TITLE
Fix spelling of errors on missung libmcount.so

### PR DESCRIPTION
**Pull request by bernhardkaindl**
Just a suggestion to fix those messages, I saw them during autotest run because of some error in starting the autotest.

The commits from Paul were here in error, removed, only the single remaining commit is what I wanted to send.

### CODE REVIEW

**Comment by namhyung** [comment]
Can you please update the commit message to contain your signed-off-by after reading [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md)?

**Comment by honggyukim** [comment]
That means please put signed-off-by as follows with your name.
```
Signed-off-by: Honggyu Kim <honggyu.kp@gmail.com>
```
`git commit --amend -s` will automatically add it if you've set your name in `.gitconfig` properly.

**Comment by bernhardkaindl** [comment]
> That means please put signed-off-by as follows with your name.
> 
> ```
> Signed-off-by: Honggyu Kim <honggyu.kp@gmail.com>
> ```
> 
> `git commit --amend -s` will automatically add it if you've set your name in `.gitconfig` properly.

Ah, thanks! done.

**Comment by honggyukim** [comment]
@bernhardkaindl Thanks. Looks good now.

File: cmds/record.c [hunk 1]
```diff
@@ -337,7 +337,7 @@ static void setup_child_environ(struct opts *opts, int argc, char *argv[])
 
 	libpath = get_libmcount_path(opts);
 	if (libpath == NULL)
-		pr_err_ns("cannot found libmcount.so\n");
+		pr_err_ns("uftrace could not find libmcount.so for record-tracing\n");
```
Comment (namhyung) [inline cmds/record.c:340]: Personally 'for recording' looks slightly better.

**Pull request merged**